### PR TITLE
Fix link to Helium 5G

### PR DESCRIPTION
**Pull request by malcolmmadsheep**

### CODE REVIEW

File: docs/faq/helium-network.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ Today, the Helium blockchain, and its hundreds of thousands of Hotspots deployed
 Hotspots enable people to own and operate a wireless network that supports a wide range of Internet of Things (IoT) devices, from smart pet collars and home delivery systems, to smart bikes and scooters, medical transport and tracking devices, cooling systems, smart lighting systems, and more. [Here’s a helpful video about how Helium works](https://www.youtube.com/watch?v=67aQeZTaqNY). 
 
 ### Can I connect my phone or computer to Hotspots?
-Not currently. The Network is designed for IoT devices that use LoRaWAN. In the future, [Helium 5G](https://www.helium.com/5g) will enable high-bandwidth devices such as smartphones to connect. 
+Not currently. The Network is designed for IoT devices that use LoRaWAN. In the future, [Helium 5G](https://www.helium.com/5G) will enable high-bandwidth devices such as smartphones to connect.
```
Comment (PharkMillups) [inline docs/faq/helium-network.mdx]:
```suggestion
Yes. As of early 2022 the community has started to roll out [4G/LTE and 5G coverage on the Helium blockchain](https://www.helium.com/5G).
```

**Comment by PharkMillups** [comment]
Thanks for the PR! I actually think we can update this to tell people that the 5g roll out has started and phones can connect. Take a look at my edits and merge if you think this makes sense.

**Comment by malcolmmadsheep** [comment]
Thanks! I've committed the updates

**Review by PharkMillups — APPROVED** [review]

**Comment by PharkMillups** [comment]
Thanks! Merging.

**Pull request merged**